### PR TITLE
chore: bump eslint-config-airbnb

### PR DESCRIPTION
**Pull request by SimenB**
Gets rid of peer dependency warning when running `yarn`. Also bumps `set-value` and `handlebars` for vulnerability fixes

### CODE REVIEW

File: src/utils/normalizeConfig.js [hunk 1]
```diff
@@ -111,13 +111,15 @@ const normalizeCliOptions = rawConfig =>
 
     const value = rawConfig[key] !== undefined ? rawConfig[key] : defaultValue;
 
+    // eslint-disable-next-line prefer-object-spread
```
Comment (ljharb) [inline src/utils/normalizeConfig.js]: it should either be disabled in the whole project (if it supports a node without spread), or it should be fixed here.

**Comment by SimenB** [comment]
hmm, some ugly config option error with eslint 4. ideas on the cleanest way to deal with it? override the config for the rule?

```
    /home/travis/build/jest-community/jest-runner-eslint/node_modules/eslint-config-airbnb-base/rules/style.js:
    	Configuration for rule "camelcase" is invalid:
    	Value {"properties":"never","ignoreDestructuring":false} should NOT have additional properties.
    Referenced from: /home/travis/build/jest-community/jest-runner-eslint/node_modules/eslint-config-airbnb-base/index.js
    Referenced from: /home/travis/build/jest-community/jest-runner-eslint/.eslintrc.json
      at validateRuleOptions (node_modules/eslint/lib/config/config-validator.js:119:19)
          at Array.forEach (<anonymous>)
          at Array.reduceRight (<anonymous>)
```

**Comment by ljharb** [comment]
We’ll have to install the older version of the airbnb config that supports eslint 4, when testing eslint 4 - ie, `npm ls` must pass before tests can be meaningful.

**Comment by SimenB** [comment]
Makes sense, thanks!

**Comment by SimenB** [comment]
This should be easier to do properly now thanks to the work of @grxy over in #83. If anyone feels up for it 😀

**Comment by MichaelDeBoey** [comment]
Closed by #92

**Comment by SimenB** [comment]
🎉

**Pull request closed without merging**